prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,71 @@ nav_section: prebid-mobile
+Prebid Mobile libraries are available for iOS and Android.
 
-### How it Works
+## Benefits and Features
 
-The Prebid Mobile SDK sends requests to Prebid Server to help app developers to access additional demand that is not available from their mobile ad network.
+Some of the benefits to using the Prebid Mobile header bidding solution include:
 
-This means you **must** have a Prebid Server account in order to use Prebid Mobile. To set up your Prebid Server account for Prebid Mobile, refer to [Get Started with Prebid Server]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html)
+-   Provides a single integration point with Prebid Server, providing direct access to more mobile buyers.
+-   Allows for server-side configuration management; no need for developers to update the app to make configuration changes.
+-   Provides a transparent, open source header bidding solution.
+-   Flattens mediation layers, reducing ad ops burden for managing mediation partners.
+-   Reduces latency compared to mediation.
+-   Designed to integrate with Prebid Server, and can also integrate with any vendor hosting their own Prebid Server deployment (vendor must be registered in Prebid Mobile as a Prebid Server host).
 
- 1. As the Prebid Mobile module runs, it fetches bids from your demand partners integrated with Prebid Server.
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+## Requirements
+
+-   **Prebid Server Configuration**  
+    You *must* have a Prebid Server account in order to use Prebid Mobile. Prebid Server is a server-based host that communicates bid requests and responses between Prebid Mobile and demand partners.  
+
+    To set up your Prebid Server account for Prebid Mobile, refer to [Getting Started with Prebid Mobile]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
+-   **Primary Ad Server Setup**  
+    Prebid Mobile enables you to retrieve bids simultaneously from multiple demand partners outside of a single ad server. This header bidding process does not determine a final winning bid. Ad ops users need to set up line items associated with each ad unit on a primary ad server. Prebid Mobile will send bids (via the primary ad server SDK) to the primary ad server. This ad server will include the bids from the header bidding process in determining a winning bid. (See "How It Works" below.)
```

Comment:
Much of this is background information.  Seems like we should be linking to the Ad Ops documentation here.  The "requirement" is that LIs and Creatives be set up in the ad server according to our documentation.